frontend/src/services/Native/index.d.ts

```diff
@@ -5,6 +5,7 @@ declare global {
     ReactNativeWebView?: ReactNativeWebView;
     NativeRobosats?: NativeRobosats;
     RobosatsSettings: 'web-basic' | 'web-pro' | 'selfhosted-basic' | 'selfhosted-pro';
+    DesktopRobosats: undefined | 'Desktop-App';
```

Comment:
maybe we can generalize this do it more similar to `RobosatsSettings`, if we call this `RobosatsClient: 'desktop-app'` will be more readable and I can do the same for Mobile in another PR, what do you think?

Reply:
i didn't understand

Reply:
Change this línea for:
```
RobosatsClient: 'desktop-app'
```
And where you are checking in other parts for `DesktopRobosats` switch it to `RobosatsClient === 'desktop-app'` that way it's more readable and will help other platforms such as the mobile android to reuse it

Reply:
ok i guess then i need to change in html file also, to window.RobosatsClient = 'desktop-app'

Reply:
and replace every use of window.DesktopRobosats === 'Desktop-App' with RobosatsClient === 'desktop-app'

Reply:
That's correct